start-all.sh

```diff
@@ -44,22 +46,40 @@ else
     PROMETHEUS_NAME=aprom-$PROMETHEUS_PORT
 fi
 
+if [ -z $NODES ]; then
+    NODES="127.0.0.1"
```

Comment:
Does that mean that you can not use a local file? if no nodes are supplied just take the configuration file